### ISSUE TITLE
Execution Block ID uid://A001/X15a0/X106 Sgr_A_st_r_03_TM1

### ISSUE THREAD

**Issue by keflavich** [ACES-CMZ/reduction_ACES#263]
Sgr_A_st_r_03_TM1
[uid://A001/X15a0/X106](https://almascience.org/aq/?result_view=observation&mous=uid://A001/X15a0/X106)

* [x] Observations completed?
* [ ] Delivered?
* [ ] Downloaded? (specify where)
  * [ ] hipergator
  * [ ] hipergator pipeline run

* [ ] [Weblog](https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_r_03_TM1/html/) unpacked
* [ ] [Weblog](https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_r_03_TM1/html/) Quality Assessment?
* [ ] Imaging: Continuum
* [ ] Imaging: Lines

  * [ ] SPW25 H13CN
  * [ ] SPW27 H13CO+/SiO
  * [ ] SPW29 HCO+
  * [ ] SPW31 HNCO
  * [ ] SPW33 Cont 1
  * [ ] SPW35 Cont 2

**Comment by keflavich** [comment]
I removed `calibrated/` today, as it was empty.  The pipeline should rerun tonight.

**Comment by keflavich** [comment]
Probably a corrupt file:
```
$ ls -lhrt raw/*uid___A002_Xfeb5e6_X4fa5*
total 744K
-rw-r--r-- 1 adamginsburg adamginsburg 711K Sep 22 21:10 Main.xml
-rw-r--r-- 1 adamginsburg adamginsburg  12K Sep 22 21:10 ASDM.xml
drwxr-xr-x 2 adamginsburg adamginsburg  20K Oct 23 05:50 ASDMBinary
```
that's it, there should be more files

**Comment by d-l-walker** [comment]
Yeah, I've encountered this exact issue (`*.ms/SYSCAL does not exist`) this week while helping a user with their data. Just had to re-download the data ...

**Comment by keflavich** [comment]
```
$ ls -lhrt 2021.1.00172.L_uid___A002_Xfeb5e6_X4fa5.asdm.sdm.tar
-rw-r--r-- 1 adamginsburg adamginsburg 71G Oct 22 20:48 2021.1.00172.L_uid___A002_Xfeb5e6_X4fa5.asdm.sdm.tar
$ md5sum 2021.1.00172.L_uid___A002_Xfeb5e6_X4fa5.asdm.sdm.tar
49c35a99c57562657ac352d2c99e658e  2021.1.00172.L_uid___A002_Xfeb5e6_X4fa5.asdm.sdm.tar
```

**Comment by keflavich** [comment]
redownloaded
```
$ ls -lhrt *X4fa5.asdm.sdm.*
-rw-r--r-- 1 adamginsburg adamginsburg 71G Oct 22 20:48 corrupted_2021.1.00172.L_uid___A002_Xfeb5e6_X4fa5.asdm.sdm.tar
-rw-r--r-- 1 adamginsburg adamginsburg 70G Dec  2 06:05 2021.1.00172.L_uid___A002_Xfeb5e6_X4fa5.asdm.sdm.tar
```

**Comment by keflavich** [comment]
I re-extracted the more recent tarball today.

```
$ md5sum 2021.1.00172.L_uid___A002_Xfeb5e6_X4fa5.asdm.sdm.tar
ff74671933cacf5cb8ca45bbc97d1f3d  2021.1.00172.L_uid___A002_Xfeb5e6_X4fa5.asdm.sdm.tar
```

**Comment by keflavich** [comment]
Line cubes are good, continuum images are junk.  Same issue noted in https://github.com/ACES-CMZ/reduction_ACES/issues/125#issuecomment-1430411878, most likely.

**Comment by keflavich** [comment]
Cubes & lines both look good; checked together on April 19 zoom call.

**Comment by keflavich** [comment]
TODO: experiment with robust=-2 because the beam is too large across the board.

**Comment by d-l-walker** [comment]
To-do: check whether the products are still size mitigated. 

This issue is marked as done, but I don't see any PR, nor do I see the updated parameters in `override_teclean_commands.json`. 

(Tagging #179 for tracking)

**Comment by pyhsiehATalma** [comment]
![截圖 2024-01-24 13 41 46](https://github.com/ACES-CMZ/reduction_ACES/assets/95929782/80ea481a-d5bb-4ba8-910f-f3716c40fb11)

continuum of spw 33 line cube was not (fully?) subtracted in the pipeline version (using the existing cube under /working). For the combination of mosaic CS(2-1) I temporary fixed this issue with imcontsub. The imcontsub was applied with a sub-cube from -300 km/s to 300 km/s. No divergence was found.

**Comment by betacygni** [comment]
**QA - Line contamination in continuum images from high/low frequencies**

New image for continuum of spw33_35 is missing in Globus

**Comment by d-l-walker** [comment]
Reminder that SPWs 25,27,29,31,35 all need to be un-size-mitigated and re-stat-cont-ed @keflavich

**Comment by keflavich** [comment]
Moved files: `mv *spw{25,27,29,31,35}* sizemitigated/`.  Rerun forthcoming.

**Comment by keflavich** [comment]
This one has caused a problem.
```
AssertionError: /orange/adamginsburg/ACES//data//2021.1.00172.L/science_goal.uid___A001_X1590_X30a8/group.uid___A001_X1590_X30a9/member.uid___A001_X15a0_X106/calibrated/working/uid___A002_Xfeb5e6_X4fa5_target_line.ms
```

```
$ ls -d /orange/adamginsburg/ACES//data//2021.1.00172.L/science_goal.uid___A001_X1590_X30a8/group.uid___A001_X1590_X30a9/member.uid___A001_X15a0_X106/calibrated/working/*ms/orange/adamginsburg/ACES//data//2021.1.00172.L/science_goal.uid___A001_X1590_X30a8/group.uid___A001_X1590_X30a9/member.uid___A001_X15a0_X106/calibrated/working/uid___A002_Xfeb5e6_X4fa5.ms
/orange/adamginsburg/ACES//data//2021.1.00172.L/science_goal.uid___A001_X1590_X30a8/group.uid___A001_X1590_X30a9/member.uid___A001_X15a0_X106/calibrated/working/uid___A002_Xfec857_X9f6.ms
```

It looks like the imaging parameters do not match the data.

**Comment by betacygni** [comment]
**QA - Line contamination in continuum images from high/low frequencies**

Summary: Could not find old image for spw25_27. New images look good, maybe a little bit too much structure in the spw25_27 which may resemble the structures that were seen in previous spw25_27 images when there was a little bit of line contamination, but it does not look wrong

Files checked:
New: spw25_27 and spw33_35
Old: v1.1_20240314_high_spw33_35

Attached image:
Top-left: spw33_35
Top-right: old_spw33_35
Bottom-left: spw25_27

<img width="904" alt="Screenshot 2024-05-08 at 14 46 34" src="https://github.com/ACES-CMZ/reduction_ACES/assets/16014764/7ac2b368-be89-4f33-a159-600ddf13a05d">
